starter-code/src/functions-and-arrays.js

```diff
@@ -11,14 +16,45 @@ var words = [
   'crackpot'
 ];
 
-// Calculating a Sum
+function findLongestWord(target){
+  var sorted = target.sort(
+    function(a, b){
+      return b.length - a.length;}
+  );
+  return sorted[0]
+};
+
+findLongestWord(words);
 
+
+// Calculating a Sum
 var numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
+var sum = 0;
+function sumArray(target){
+  var result = target.reduce(function(accumulator, currentValue, currentIndex, array) {
+    return accumulator + currentValue;
+  });
+  sum = result;
+  return sum;
+}
+
+sumArray(numbers)
+
+
 // Calculate the Average
 
 var numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
 
+function averageNumbers(target){
+  sumArray(target);  //get the sum
```

Comment:
Same here you should add a condition, so when target is empty you return undefined